xbmc/rendering/gl/RenderSystemGL.cpp

```diff
@@ -55,14 +55,22 @@ bool CRenderSystemGL::InitRenderSystem()
       GLint i;
       for (i = 0; i < n; i++)
       {
-        m_RenderExtensions += (const char*)glGetStringi(GL_EXTENSIONS, i);
-        m_RenderExtensions += " ";
+        auto extensions = (const char*) glGetStringi(GL_EXTENSIONS, i);
+        if (extensions)
```

Comment:
that check is pretty useless because we already queried the number of extensions. hence glGetStringi must return an extension at the given index.

Reply:
I agree, just wanted to be safe. We can try without and see if that fixes the issues on Fedora.